openedx/core/djangoapps/notifications/templates/notifications/digest_footer.html

```diff
@@ -0,0 +1,48 @@
+<table cellpadding="0" cellspacing="0" style="padding:1.875rem; background-color:#F2F0EF; color:black; font-weight:400; font-size:0.75rem;line-height:20px" width="100%">
+    <tbody>
+        <tr>
+            <td>
+                <table width="100%">
+                    <tbody>
+                        <tr>
+                            <td width="20%" align="left">
+                                <img src="{{ logo_url }}" style="width: auto;" height="24" alt="Logo"/></a>
+                            </td>
+                            <td width="80%" align="right">
+                                <table>
+                                    <tbody>
+                                        <tr>
+                                            {% for social_name, social_data in social_media.items %}
+                                                <td>
+                                                    <a href="{{social_data.url}}">
+                                                        <img src="{{social_data.icon}}" style="max-height: 24px; max-width: 24px" alt="{{social_name}}" />
+                                                    </a>
+                                                </td>
+                                            {% endfor %}
+                                        </tr>
+                                    </tbody>
+                                </table>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                    You are receiving this email because you have subscribed to email digest
+                </p>
+                <p>
+                    <a href="{{notification_settings_url}}" rel="noopener noreferrer" target="_blank" style="color: black">
+                        Notification Settings
+                    </a>
+                </p>
+                <p>
+                    &copy; 2023 edX LLC. All Rights Reserved <br/>
+                    7900 Harkins Road, Lanham MD 20706
+                </p>
+            </td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
how does this footer compare to other ace footers in platform? Is it similar, or designed completely from scratch?

Reply:
This footer is designed from scratch.